auth.config.ts

```diff
@@ -5,14 +5,15 @@ export const authConfig = {
   pages: {
     signIn: '/login',
     newUser: '/signup'
-  },
+    },
```

Comment:
remove this unrequired indentation change.

Reply:
done!